packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -1,9 +1,25 @@
+import * as _ from 'lodash';
+
 import { BigNumber, ERC20BridgeSource, SignedOrder } from '../..';
-import { getCurveInfo, isCurveSource } from '../source_utils';
 
-import { DEFAULT_FAKE_BUY_OPTS } from './constants';
+import { DEFAULT_FAKE_BUY_OPTS, MAINNET_CURVE_CONTRACTS } from './constants';
+import {
+    BalancerPool,
+    computeBalancerBuyQuote,
+    computeBalancerSellQuote,
+    getBalancerPoolsForPairAsync,
+} from './balancer_utils';
+import { getCurveAddressesForPair } from './curve_utils';
 import { getMultiBridgeIntermediateToken } from './multibridge_utils';
-import { BatchedOperation, DexSample, FakeBuyOpts } from './types';
+import {
+    BalancerFillData,
+    BatchedOperation,
+    CurveFillData,
+    DexSample,
+    FakeBuyOpts,
+    SourceQuoteOperation,
+    UniswapV2FillData,
+} from './types';
 
 /**
```

Comment:
Starting to feel like this needs to be a class so it can hold state like registry addresses and cached balancer pools and calls to some of these ops can be more legible. I think it'll clean up some of the tests as well.